ext/http/02_websocket.ts

```diff
@@ -91,7 +91,7 @@ function upgradeWebSocket(request, options = { __proto__: null }) {
   const socket = createWebSocketBranded(WebSocket);
   setEventTargetData(socket);
   socket[_server] = true;
-  socket[_idleTimeoutDuration] = options.idleTimeout ?? 120;
+  socket[_idleTimeoutDuration] = options.idleTimeout ?? 30;
```

Comment:
Can you add a comment? Maybe something like: "30s works with common Ngnix setups. See issue: ..."